README_LINUX.md

```diff
@@ -296,7 +296,7 @@ environment variables:
    $> export SC_JACK_DEFAULT_INPUTS="system"
    ```
 
- * SC_JACK_DEFAULT_OUTPUTS comma-separated list of jack ports that the server's outputs should be connected to by default.
+ * SC_JACK_DEFAULT_OUTPUTS comma-separated list of jack ports that the server's outputs should be connected to by defaul>t.
```

Comment:
It seems there's an accidental addition here ("defaul>t")

Reply:
Just fixed it. Thanks for spotting it.